README.md

```diff
@@ -1,2 +1,383 @@
-# year-cohort-projectName
-project template repo
+# Restaurant API
+
+## Resources
+MongoDB.
+
+Node.js.
+
+Express.js.
+
+RESTful API design.
+
+
+### Collection Structure
+##### Collection structure for restaurant.starters, restaurant.mains, restaurant.desserts, restaurant.refreshments
+
+| Column 1       | Column 2     | Column 3 |
+| :------------- | :----------: | :--------:|
+| _id |  Unique menu item id, auto-generated  | String |
+| name   | Name of dish | String |
+| description   | Description of the dish | String |
+| price   | Amount of money the dish costs | Decimal128 |
+
+##### Collection structure for orders
+
+| Column 1       | Column 2     | Column 3 |
+| :------------- | :----------: | :--------:|
+| _id |  Unique order id, auto-generated  | String |
+| name   | Name of customer | String |
+| deliveryAddress   | Address of customer where order needs to be delivered | String |
+| email   | Customers email | String |
+| isOrderSubmitted | Has the customer submitted final order | Bool |
+| timePlaced   | Time order was submitted by customer | Date |
+| orderItems   | Holds object of each menu item id and quantity that customer has ordered | Arrray |
+
+## API Calls
+
+### GET
+
+#### Getting items on the menu
+
+Returns a JSON string of dishes (option for all dishes, and dishes by type). No authentication required.
+
+
+`GET /dishes/starters`  //returns starters
+
+
+`GET /dishes/mains`  //returns mains
+
+
+`GET /dishes/desserts`  //returns desserts
+
+
+`GET /dishes/refreshments`  //returns refreshments
+
+
+##### Sample Call
+	
+	fetch('http://localhost:3000/dishes')
+		.then (res => res.json())
+		.then ((data) => {
+			//do stuff with your data
+			})
+   	 })
+
+##### Success Response
+	
+	{
+		"success": true,
+		"message": "Requested dishes successfully retrieved.",
+		"status": 200,
+		"data": [{}, {}, {}]
+	}
+	
+##### Error Response
+
+	{
+		"success": false,
+		"message": "The resources requested do not exist at the desired location.",
+		"status": 404
+	}
+	
```

Comment:
Please delete from line 81 and up - this will conflict with out README and will save you some pain